<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaoHang</class>
 <widget class="QWidget" name="DaoHang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园寻路系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="School.qrc">
    <normaloff>:/School/Picture/tb.png</normaloff>:/School/Picture/tb.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>491</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>起点：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="ShengCheng">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="3">
     <widget class="QComboBox" name="ZhongDian">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="YuYin_Button">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>语音播报</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>终点：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="FangShi">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QComboBox" name="QiDian">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>寻路系统</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="TingZhi_Button">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>停止播报</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="DaoHang_Edit">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>320</y>
     <width>471</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>770</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>当前用户：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="CurrentYH">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="School.qrc"/>
 </resources>
 <connections/>
</ui>
